src/Storages/PostgreSQL/StorageMaterializedPostgreSQL.cpp

```diff
@@ -195,7 +197,8 @@ void StorageMaterializedPostgreSQL::createNestedIfNeeded(PostgreSQLTableStructur
         const auto ast_create = getCreateNestedTableQuery(std::move(table_structure), table_override);
         auto table_id = getStorageID();
         auto tmp_nested_table_id = StorageID(table_id.database_name, getNestedTableName());
-        LOG_DEBUG(log, "Creating clickhouse table for postgresql table {}", table_id.getNameForLogs());
+        LOG_DEBUG(log, "Creating clickhouse table for postgresql table {} (ast: {})",
+                  table_id.getNameForLogs(), serializeAST(*ast_create));
```

Comment:
BTW, we have `IAST::formatForLogging` that formats ast with settings suitable for logging (oneline, hide secrets, etc)
https://github.com/ClickHouse/ClickHouse/blob/efb82a93c5fd365caf01a850cb90511153e9fa7d/src/Parsers/IAST.h#L283

Doesn't make to much difference here, but just in case.